src/app/shell/personal-cabinet/provider/create-achievement/create-achievement.component.ts

```diff
@@ -59,12 +53,24 @@ export class CreateAchievementComponent implements OnInit, OnDestroy {
 
   ngOnInit(): void {
     this.workshopId = this.route.snapshot.paramMap.get('param');
-    this.store.dispatch(new GetWorkshopById(this.workshopId));
+    this.store.dispatch([
+      new GetWorkshopById(this.workshopId), 
+      new GetChildrenByWorkshopId(this.workshopId)
+    ]);
+
     this.workshop$
     .pipe(
       takeUntil(this.destroy$),
       filter((workshop) => !!workshop)
-    ).subscribe((workshop: Workshop) => this.workshop = workshop);      
+    ).subscribe((workshop: Workshop) => {
+      this.workshop = workshop;
+    });   
+    
+    this.approvedChildren$
+    .pipe(
+      takeUntil(this.destroy$),
+      filter((approvedChildren) => !!approvedChildren)
+    ).subscribe((approvedChildren: Child[]) => this.approvedChildren = approvedChildren);
```

Comment:
you can add here a mapping for applying util. get Full Name, and create new array that have only childId and Full Child Name, try to implement this approach